### DISCUSSION TITLE
wiki/build_web_road/optblog/test_staller

### DISCUSSION THREAD

**Discussion by giscus** [haoshuaifei/blog-comments#5] [Announcements]
# wiki/build_web_road/optblog/test_staller

彩色代码块设置 child:codeblock 并设置 color:颜色枚举 可以实现 10 种不同颜色的代码块，彩色代码块一般可以用在代码正确与错误的示范对比场景。      推荐的写法123func test() &#123;    // ...&#125;               不推荐的写法123func test() -> () &#123;    // ...&#125;

http://localhost:4000/wiki/build_web_road/optblog/test_staller.html

**Comment by haoshuaifei** [comment]
测试一下
